### PR TITLE
Add backwards-compat imports with deprecation warnings

### PR DESCRIPTION
**Pull request by NeonDaniel**
Addresses failures: https://github.com/NeonGeckoCom/NeonCore/pull/599

### CODE REVIEW

**Comment by JarbasAl** [comment]
dev branch already is on 0.1.0a1

last stable release should have all th deprecation warnings, this would need to be a manual post release

**Comment by NeonDaniel** [comment]
> dev branch already is on 0.1.0a1
> 
> last stable release should have all th deprecation warnings, this would need to be a manual post release

Rebased on `master` and manually specified a post version

File: ovos_utils/messagebus.py [hunk 1]
```diff
@@ -7,14 +7,38 @@
 from pyee import BaseEventEmitter
 
 from ovos_utils import create_loop
-from ovos_utils.json_helper import merge_dict
 from ovos_utils.log import LOG, log_deprecation, deprecated
+from ovos_utils.events import EventContainer as _EC
```
Comment (JarbasAl) [inline ovos_utils/messagebus.py:11]: this adds a circular import

```
Traceback (most recent call last):
  File "/home/miro/PycharmProjects/ovos-core/ovos_core/__main__.py", line 25, in <module>
    from ovos_bus_client.util.scheduler import EventScheduler
  File "/home/miro/PycharmProjects/ovos-bus-client/ovos_bus_client/util/__init__.py", line 26, in <module>
    from ovos_bus_client.util.scheduler import EventScheduler
  File "/home/miro/PycharmProjects/ovos-bus-client/ovos_bus_client/util/scheduler.py", line 34, in <module>
    from ovos_utils.events import EventContainer as _EventContainer
  File "/home/miro/PycharmProjects/ovos-utils/ovos_utils/events.py", line 6, in <module>
    import ovos_utils.messagebus
  File "/home/miro/PycharmProjects/ovos-utils/ovos_utils/messagebus.py", line 11, in <module>
    from ovos_utils.events import EventContainer as _EC
ImportError: cannot import name 'EventContainer' from partially initialized module 'ovos_utils.events' (most likely due to a circular import) (/home/miro/PycharmProjects/ovos-utils/ovos_utils/events.py)
```

File: ovos_utils/messagebus.py [hunk 2]
```diff
@@ -7,14 +7,38 @@
...
+    return create_wrapper(*args, **kwargs)
+
+
+def get_handler_name(*args, **kwargs):
+    log_deprecation("Import from `ovos_utils.events`", "0.1.0")
+    from ovos_utils.events import get_handler_name
+    return get_handler_name(*args, **kwargs)
+
+
+def merge_dict(*args, **kwargs):
```
Comment (JarbasAl) [inline ovos_utils/messagebus.py:36]: this was never a valid import from here ?

**Pull request merged**